src/main/java/de/zalando/aruha/nakadi/controller/EventStreamController.java

```diff
@@ -82,17 +80,11 @@ public StreamingResponseBody streamEvents(
             @Nullable @RequestParam(value = "stream_timeout", required = false) final Integer streamTimeout,
             @Nullable @RequestParam(value = "stream_keep_alive_limit", required = false) final Integer streamKeepAliveLimit,
             @Nullable @RequestHeader(name = "X-nakadi-cursors", required = false) final String cursorsStr,
-            final NativeWebRequest request, final HttpServletResponse response) throws IOException {
+            final HttpServletRequest request, final HttpServletResponse response) throws IOException {
```

Comment:
@antban is it fine if I directly inject HttpServletRequest? (Not take it from NativeWebRequest)